<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>读入路径</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="filePathBox"/>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>存储路径</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="filePathBox2"/>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_2">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>输入：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>h</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>W</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>L</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>y</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>U</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEdit_5"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>温馨提示：请不要同时输入无穷远光和有限远光参数</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>提示：
无穷远光：
输入入瞳半径h和视场角W

有限远光：
输入物距L， 物高y，物方孔径角U


温馨提示：
理想像高都是d光理想像高</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>色光</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="Light">
                <property name="maxVisibleItems">
                 <number>10</number>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string/>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Light_d</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Light_f</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Light_c</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>光线追踪       </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="result">
                <item>
                 <property name="text">
                  <string/>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>透镜固定参数</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>理想像距</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>实际像位置</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>理想像高</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>实际像高</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>各类像差</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="aberration">
                <item>
                 <property name="text">
                  <string notr="true"/>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>球差</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>场曲像散</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>畸变</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>色差</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>彗差</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Show</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="file_output_btn">
              <property name="text">
               <string>File Output</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonclose">
              <property name="text">
               <string>Quit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>copyright@2019 By 胡启笠 周皓宇 张旭</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonclose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>858</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
